uaclient/tests/test_event_logger.py

```diff
@@ -4,14 +4,15 @@
 
 import mock
 import pytest
+import yaml
 
 from uaclient.event_logger import JSON_SCHEMA_VERSION, EventLoggerMode
 
 
 class TestEventLogger:
     @pytest.mark.parametrize(
         "event_mode",
-        ((EventLoggerMode.CLI), (EventLoggerMode.MACHINE_READABLE)),
+        ((EventLoggerMode.CLI), (EventLoggerMode.JSON), EventLoggerMode.YAML),
     )
     def test_process_events(self, event_mode, event):
```

Comment:
Should we add a test here were we set the command as `status`  and give an `output` to the `set_output_status` ?

Reply:
added